algotaurus.py

```diff
@@ -769,7 +818,10 @@ def execute_code(self):
             self.linebox.delete(current_pos)
             self.linebox.configure(state='disabled')
             if not self.stop:
-                self.tkMessageBox.showinfo('Result', result)
+                if robot.success:
+                    self.tkMessageBox.showinfo(_('Result'), result)
+                else:
+                    self.tkMessageBox.showinfo(_('Result'), result, icon='error')
```

Comment:
Modify outline of the message window (warning or info) based on the robot's success. (+missing translation)